public/checkout.js

```diff
@@ -192,7 +202,11 @@ var packeteryLoadCheckout = function( $, settings ) {
 			}
 		};
 
-		updateWidgetButtonVisibility( getShippingRateId() );
+		var isPickupPointChosen = function ($widgetDiv) {
+			return $widgetDiv.find('.packeta-widget-info').html() !== '';
```

Comment:
Pokud jQuery element nenajde, tak funkce isPickupPointChosen vrátí true. Prosím upravit.

Reply:
Upravil jsem to, možná je to trochu zbytečně defensivní.